client/src/components/calendar/Demo.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import Paper from '@material-ui/core/Paper';
+import { ViewState } from '@devexpress/dx-react-scheduler';
+import {
+  Scheduler,
+  WeekView,
+  Appointments,
+  Toolbar,
+  ViewSwitcher,
+  MonthView,
+  DayView,
+} from '@devexpress/dx-react-scheduler-material-ui';
+
+import { appointments } from './month-appointments';
+import { DemoProps, DemoState } from '../../types/interfaces';
```

Comment:
Change DemoProps and state to a more appropriate name, same with the component (ex.. GoogleCalendarScheduler)